extensions/wikia/RTE/RTE.class.php

```diff
@@ -61,6 +61,39 @@ public static function replacePlaceholder($var) {
 		$data = RTEData::get('placeholder', intval($var[1]));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "placeholder" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)